scripts/searchSamadi.py

```diff
@@ -0,0 +1,20 @@
+import commands, sys
+
+stringToLookFor = "*v0.1.0+76X_HHAnalysis_2016-04-04.v0"
+output = commands.getoutput("search_SAMADhi.py sample -n %s"%stringToLookFor)
```

Comment:
`search_SAMADhi` can be loaded and called directly, no need to pass by command line